.github/workflows/chatgpt-review.yml

```diff
@@ -0,0 +1,37 @@
+name: ChatGPT
+
+on:
+  pull_request:
+    types: [opened, synchronize]
+
+jobs:
+
+  code_review_job:
+    runs-on: ubuntu-latest
+    name: ChatGPT Code Review
+
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+
+      - name: AI Code Reviewer
+        uses: hohyon-ryu/ai-codereviewer@code_review
+        with:
+          GITHUB_TOKEN: ${{ secrets.CUSTOM_GITHUB_TOKEN }} # The GITHUB_TOKEN is there by default so you just need to keep it like it is and not necessarily need to add it as secret as it will throw an error. [More Details](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#about-the-github_token-secret)
+          OPENAI_API_KEY: ${{ secrets.OPENAI_TOKEN }}
```

Comment:
The change from `OPENAI_API_KEY` to `OPENAI_TOKEN` might cause confusion. Please ensure that the correct key is being used and that the repository secrets are updated accordingly.